tensorflow/lite/micro/fake_micro_context.cc

```diff
@@ -39,10 +39,19 @@ FakeMicroContext::FakeMicroContext(TfLiteTensor* tensors,
       tensors_(tensors),
       allocator_(allocator) {}
 
-TfLiteTensor* FakeMicroContext::GetTensor(int tensor_index) {
+TfLiteTensor* FakeMicroContext::AllocateTempTfLiteTensor(int tensor_index) {
+  allocated_tensor_count_++;
   return &tensors_[tensor_index];
 }
 
+void FakeMicroContext::DeallocateTempTfLiteTensor(TfLiteTensor* tensor) {
+  allocated_tensor_count_--;
+}
+
+bool FakeMicroContext::IsAllTempTfLiteTensorDeallocated() {
```

Comment:
Maybe change method name to AreAllTempTfLiteTensorsDeallocated (using plural), or just AllTempTfLiteTensorsDeallocated?

Reply:
Just saw this comment. Will make a fixit bug for it. I am not sure of using Are. I might change it to HasAllocatedTempTfLiteTensor(), which avoid is vs are